modules/local/gridss/index/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: gridss_index
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::gridss=2.13.2=hdfd78af_3
```

Comment:
can you remove the build here. It can cause issues when running with conda on different systems

Reply:
This is required as earlier builds are broken:

- https://github.com/bioconda/bioconda-recipes/pull/45641

GRIDSS is also only available for the 'linux-64' platform, so this should be safe in that regard for users.

And thanks for bringing the build version to my attention - it was incorrect but I've now corrected it in https://github.com/nf-core/oncoanalyser/pull/72.